### PR TITLE
[Odie] Apply new help center redesign for Wapuu

### PR DESCRIPTION
**Pull request by AllTerrainDeveloper**
Related to: https://github.com/Automattic/ai-services/issues/555

This set of changes are intended to implement @mattrwalker changes for the Help Center integration with Odie. This is still a work in progress chain of Pull Requests, follow up pull requests will follow.

## Proposed Changes

- Added avatars for the conversation
- Moved the Odie code in Help-Center content to its own component
- Added new look and feel for the chat messages
- Added new look and feel for the input messages
- Added new button to jump to the most recent messages
- Added EllipsisMenu in the header with options (new chat, send transcript)
- Added new way of displaying the feedback for each message
- Cleaned **A LOT** of unused code
- Added some translations

## Testing Instructions

The way to test this is basically append to the URL the flag wapuu, eg: `http://...?flags=wapuu`

1. Click on the ? icon in the master bar, located in the top right corner.
2. Click on Still need help?
3. A new and fancy chat should open
4. Test the behaviour of the following things
    - Jump to Recent button, works as desired (it appears when we are not in the bottom of the conversation, it takes you to the bottom of the conversation when clicked
    - Minimise icon will hide everything and nothing will remain visible (a part for the minimised Help Center bar) 
    - Typing long texts will move the Jump To Recent button upper (just type a few `shift-enter` characters)
5. The Send message feature is out of scope for this pull request, that's a follow up
6. Clear conversation and send transcript will be also a follow up pull request

See video for easier verification:
https://github.com/Automattic/wp-calypso/assets/5689927/b839f9a1-7fd7-4e16-b380-02062ab3b96e

## Pre-merge Checklist

- [ ] Has the general commit checklist been followed? (PCYsg-hS-p2)
- [ ] https://wpcalypso.wordpress.com/devdocs/docs/testing/index.md for your changes?
- [ ] Have you tested the feature in Simple (P9HQHe-k8-p2), Atomic (P9HQHe-jW-p2), and self-hosted Jetpack sites (PCYsg-g6b-p2)?
- [ ] Have you checked for TypeScript, React or other console errors?
- [ ] Have you used memoizing on expensive computations? More info in [Memoizing with create-selector](https://github.com/Automattic/wp-calypso/blob/trunk/packages/state-utils/src/create-selector/README.md) and [Using memoizing selectors](https://react-redux.js.org/api/hooks#using-memoizing-selectors) and [Our Approach to Data](https://github.com/Automattic/wp-calypso/blob/trunk/docs/our-approach-to-data.md)
- [ ] Have we added the "[Status] String Freeze" label as soon as any new strings were ready for translation (p4TIVU-5Jq-p2)?
- [ ] For changes affecting Jetpack: Have we added the "[Status] Needs Privacy Updates" label if this pull request changes what data or activity we track or use (p4TIVU-ajp-p2)?

### CODE REVIEW

File: client/odie/context/initial-messages.ts [hunk 1]
```diff
@@ -3,7 +3,42 @@ import type { Message } from '../types';
...
+			{
+				type: 'message',
+				content: 'Here some sources to learn more about WordPress',
+				role: 'bot',
+				sources: [
+					'[WordPress.com Support](https://en.support.wordpress.com/)',
+					'[WordPress.org Support Forums](https://wordpress.org/support/)',
+				],
+			},
+		];
```
Comment (AllTerrainDeveloper) [inline client/odie/context/initial-messages.ts:41]: This is mocked data in order to test the new UX. It will be eventually deleted.

**Review by dereksmart — APPROVED** [review]
Tested ok for me. No regressions found without the flag. 

Nice job cleaning out all the unused code. 

> Added new button to jump to the most recent messages

It could also be triggered when you write a new message. I tested writing a new message when midway scrolled and I was expected to be taken down. 

I also expected some feedback when interacting with the 👍 👎 

These are likely already planned to be handled in upcoming PRs so let's not worry about it here. Small PRs would be great. 

Not sure what's up with the e2e tests but they look unrelated.

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/9483605

Some locales (Hebrew, Japanese) have been temporarily machine-translated due to translator availability. All other translations are usually ready within a few days. Untranslated and machine-translated strings will be sent for translation next Monday and are expected to be completed by the following Friday.

Hi @AllTerrainDeveloper, could you please edit the description of this PR and add a screenshot for our translators? Ideally it'd include all of the following strings:
- `Start a New Chat`
- `Send Transcript to Email`
- `User profile display picture`
- `%(botName)s profile picture`
- `Loading state, awaiting response from %(botName)s`
- `Icon to expand or collapse %(botName)s messages`
- `Thumbs up icon`
- `Arrow icon`
- `Sources:`
- `Jump to recent`
- `Ask your question`

Thank you in advance!

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/9483605).

**Pull request merged**